### ISSUE TITLE
Python Error when clicking with Copy Latitude, Longitude

### ISSUE THREAD

**Issue by ciaranevans** [NationalSecurityAgency/qgis-latlontools-plugin#34]
Hi there,

I'm currently using the tool and have set the output to be MGRS.

When I click on the canvas I receive the following Python Error:

```
An error has occurred while executing Python code: 

TypeError: QgsMapToolEmitPoint.toMapCoordinates(): arguments did not match any overloaded call:   overload 1: argument 1 has unexpected type 'QgsPointXY'   overload 2: argument 1 has unexpected type 'QgsPointXY' 
Traceback (most recent call last):
  File "C:/Users/evansc/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\latlontools\copyLatLonTool.py", line 198, in clicked
    pt = self.snappoint(pt)
  File "C:/Users/evansc/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\latlontools\copyLatLonTool.py", line 153, in snappoint
    return self.toMapCoordinates(point)
TypeError: QgsMapToolEmitPoint.toMapCoordinates(): arguments did not match any overloaded call:
  overload 1: argument 1 has unexpected type 'QgsPointXY'
  overload 2: argument 1 has unexpected type 'QgsPointXY'


Python version: 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)] 
QGIS version: 3.4.0-Madeira Madeira, 4a4b62ed19 
```

**Comment by hamiltoncj** [comment]
@ciaranevans 
Sorry about that. I deleted version 3.3.3 from the QGIS repository and just pushed out version 3.3.4. It may take up to a half an hour before QGIS will report that it is there, but this problem should now be fixed. Let me know if you see any other issues.

Thanks!

**Comment by hamiltoncj** [comment]
Given that others have successfully used the fixed version and having tested this problem myself, I think it has been solved with version 3.3.4 so I am closing the ticket. If you find any other problem or have a suggestion please let me know because it helps to make the plugin even more useful to the community.
